src/saveToCase.js

```diff
@@ -40,6 +42,10 @@ define([
         return mug ? mug.p.use_index : false;
     }
 
+    function attachmentCase(mug) {
+        return mug ? mug.p.use_attachment : false;
```

Comment:
Well that was embarassing. Not sure what came over me.

I'm keeping `date_modified` and similar items as that's how they appear in the case xml. Do you think that is reasonable. I'm only keeping them there where there is a one-to-one mapping from vellum to xml